### DISCUSSION TITLE
PRs: display changes (+x/-y) in the list view?

### DISCUSSION THREAD

**Discussion by paul-marechal** [community/community#21726] [New to GitHub]
<p>Hi,</p>
<p>Did not really know where to post/ask this, basically a feature request:</p>
<p>To be able to see the amount of changes of each PR in the list view.</p>
<p>Use-case: I am running builds, and all I have to do is either go grab a coffee, or look through PRs to do reviews.</p>
<p>Problem is that I don’t necessarly have the time to review +700/-200 patches while my workstation is busy for 10 to 25min. But changes around +/-100 can entirely be reviewed in this time.</p>
<p>TBF that’s not a killer feature, it is more related to my laziness/business-mix when it comes to reviewing PRs.</p>
<p>In my case, it would give me a bit more information and comfort as a maintainer/contributor as to what level of commitment will be required to diligently do a review. Of course this is not a 100% accurate metric, but it helps.</p>
<p>If one of the concerns is that people then might avoid even looking at big patches if they can see it from the list, maybe just making it an opt-in feature so that only people wanting to use that could benefit from it, without having too bad repercussions on the whole user base behavior.</p>
<p>But then maybe I’m not using GH correctly, and I should just open PRs one by one and ctrl+w if too big.</p>

**Comment by mpboom** [comment]
<p>This is indeed the right place to post feature requests, as well! It should be logged by the moderation team soon.</p>

**Comment by MChevy422** [comment, marked as answer]
<p><strong>Hello <span><a class="mention" href="/marechal-p">@marechal-p</a></span>, <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your valued and much appreciated input.</strong>** <br><br>-MChevy422**</p>

**Comment by paul-marechal** [comment]
<p>Thank you, I just wanted to know if other people found that this would be useful too.</p>
<p>BTW, I clicked the “accept as solution” button, but I think I should have not :slight_smile:</p>
<p>Next time I’ll know!</p>

**Comment by Avasam** [comment]
Related: https://github.com/orgs/community/discussions/17275 and https://github.com/orgs/community/discussions/78897
